ProjFS.Mac/Scripts/Build.sh

```diff
@@ -14,5 +14,10 @@ PROJFS=$SRCDIR/ProjFS.Mac
 
 xcodebuild -configuration $CONFIGURATION -workspace $PROJFS/PrjFS.xcworkspace build -scheme PrjFS -derivedDataPath $ROOTDIR/BuildOutput/ProjFS.Mac/Native || exit 1
 
+# If we're building the Profiling(Release) configuration, remove Profiling() for building .NET code
+if [ "$CONFIGURATION" == "Profiling(Release)" ]; then
```

Comment:
Use `DOTNETCONFIGURATION` for consistency?

Do we need a change in the MirrorProvider scripts too?

Reply:
We do need a change in the MirrorProvider, adding that. On making that change, I discovered that the MirrorProvider's script does something different (doesn't bother making a DOTNETCONFIGURATION variable so I'll make that change in the BuildGVFSForMac script too.